ql-mbim-core.c

```diff
@@ -46,9 +46,9 @@ static int log_printf(int lvl, const char *log_msg)
     } while (0)
 
 static int file_get_value(const char *fpath, int base);
-static int flash_mode_check(void);
+int flash_mode_check(void);
 
```

Comment:
flash_mode_check option is not required by modemfwdupd for now and this function can remain static.